### ISSUE TITLE
can you declare different locations other than /docs and /config

### ISSUE THREAD

**Issue by jmazzitelli** [giantswarm/crd-docs-generator#91]
To use this tool, the git repository needs two top-level directories defined (`config` which contains `config/crd`, and `docs` which contains `docs/cr`).

Is there anyway to override this? I would like to be able to have a single top level directory with everything in it. e.g.

```
crd-docs
      |____config
      |        |___crd
      |____docs
             |____cr
```

So I can still have the `config/crd` and `docs/cr` directories, but they are all under a single top level directory `crd-docs`.

Workaround I am going to use is create softlinks in my git repo so my top level `config` and `docs` directories are linked to the corresponding directories under `crd-docs`. But I would prefer not to have these extra directories at the top of my repo.

**Comment by jmazzitelli** [comment]
I have submitted a PR that addresses this problem - please see https://github.com/giantswarm/crd-docs-generator/pull/94

cc @marians would you be able to review that PR and see if it is acceptable?

**Comment by marians** [comment]
I looked into your PR. First of all: I'm happy to see that you find crd-docs-generator worthy to pick this up and contribute.

Regarding the solution you are proposing, I'm thinking we can do better.

The old hard-coded paths for CRD and CR examples YAML were very specific when we used crd-docs-generator only at Giant Swarm and only with one source repository. Now that we at GS have split up our CRDs into multiple repositories, this is already a strange restriction.

I think we should instead be able to configure these paths independently, per source repository. And we should probably be able to configure any number of paths for both CRDs and CR examples.

The config file could look somewhat like this (excerpt):

```yaml
source_repositories:
  - url: https://github.com/giantswarm/apiextensions-application
    organization: giantswarm
    short_name: apiextensions-application
    commit_reference: v0.1.0
    crd_paths:
      - config/crd
    cr_paths:
      - docs/cr
```

Would that make sense for your use case?

**Comment by jmazzitelli** [comment]
> Would that make sense for your use case?

@marians yes! that would be great. If you implement that, I would be happy to use that feature. You can close this PR if that's the case.

**Comment by marians** [comment]
@jmazzitelli Please try #95 . Thank you!

**Comment by jmazzitelli** [comment]
> @jmazzitelli Please try #95 . Thank you!

Works perfect! Thanks! Here's what I tested with - https://github.com/jmazzitelli/kiali-operator/blob/crd-gen-fix-root-dir-2/crd-docs/config/apigen-config.yaml

Let me know when you release this and the docker image is available for use.

**Comment by marians** [comment]
Released as v0.9.0
